course_project/14/src/main/java/com/homework/twitter/TwitterController.java

```diff
@@ -0,0 +1,52 @@
+package com.homework.twitter;
+
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/twitter")
+public class TwitterController {
```

Comment:
This is not a good idea, you should have user controller and tweets controller

Twitter is the app, for each resource you should have separate controllers